elementor/elementor
# Fix: Ensure default experiment dependencies are checked

## PR Checklist



## PR Type
What kind of change does this PR introduce?


## Summary

This PR can be summarized in the following changelog entry:

* Fix: Ensure default experiment dependencies are checked

## Description
An explanation of what is done in this PR

* The experiment default state is set by developers, but can be altered by the state of a site. This fix verifies that the default state follows the restrictions for dependencies

## Test instructions
This PR can be tested by following these steps:

* Set Flexbox Container experiment default state to inactive
* Set Grid Container experiment default state to active
* Load the feature settings window
* Grid Container feature should be disabled, because one of it's dependencies (Flexbox Container) is disabled

## Quality assurance


Fixes # ED-16146
